STMicroelectronics/STM32L4_L4+/app/az_ulib_dm/src/az_ulib_registry_interface.c

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license.
+// See LICENSE file in the project root for full license information.
+
+#include "_az_ulib_registry_interface.h"
+#include "az_ulib_capability_api.h"
+#include "az_ulib_descriptor_api.h"
+#include "az_ulib_ipc_api.h"
+#include "az_ulib_registry_api.h"
+#include "az_ulib_result.h"
+#include "registry_1_model.h"
+
+#include <stddef.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+static az_result registry_1_get_value_concrete(
+    az_ulib_model_in model_in,
```

Comment:
>az_ulib_model_in model_in [](http://example.com/codeflow?start=4&length=25)

use `const registry_1_get_model_in* const in` instead of `az_ulib_model_in model_in` here

Reply:
Same for all concrete functions.